src/main/java/org/jenkinsci/test/acceptance/plugins/gitlab_plugin/GitLabServerConfig.java

```diff
@@ -0,0 +1,26 @@
+package org.jenkinsci.test.acceptance.plugins.gitlab_plugin;
+
+import jakarta.inject.Inject;
+import org.jenkinsci.test.acceptance.po.Jenkins;
+import org.jenkinsci.test.acceptance.po.PageAreaImpl;
+import static org.jenkinsci.test.acceptance.Matchers.hasContent;
+
+public class GitLabServerConfig extends PageAreaImpl {
+
+    @Inject
+    public GitLabServerConfig(Jenkins jenkins) {
+        super(jenkins, "/io-jenkins-plugins-gitlabserverconfig-servers-GitLabServers");
+    }
+
+    public void configureServer(String url) {
+        find(by.path("/io-jenkins-plugins-gitlabserverconfig-servers-GitLabServers/servers/name")).sendKeys("servername");
+        find(by.path("/io-jenkins-plugins-gitlabserverconfig-servers-GitLabServers/servers/serverUrl")).clear();
+        find(by.path("/io-jenkins-plugins-gitlabserverconfig-servers-GitLabServers/servers/serverUrl")).sendKeys(url);
+
+        find(by.option("GitLab Personal Access Token")).click();
```

Comment:
this presumably causes some dynamic javascript - so you should probably wait to ensure that has completed before you move on?   
do you not need to do anything here - like specifiy the credential?

Reply:
modification coming soon for waiting 
>do you not need to do anything here - like specifiy the credential?

No, there is only one type (GitLab API token) that can be put